### PR TITLE
Improvements to the documentation

### PR DESCRIPTION
**Pull request by zanieb**

### CODE REVIEW

File: docs/pip/environments.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ installation's environment. Unlike `pip`, uv requires using a virtual environmen
 
 uv supports creating virtual environments:
 
-```bash
-# Create a virtual environment at `.venv`
+```console
+$ # Create a virtual environment at `.venv`
 uv venv
```
Comment (eth3lbert) [inline docs/pip/environments.md:16]:
``` suggestion
$ # Create a virtual environment at `.venv`
$ uv venv
```

File: docs/pip/environments.md [hunk 2]
```diff
@@ -11,22 +11,22 @@ installation's environment. Unlike `pip`, uv requires using a virtual environmen
...
 ```
 
 A specific name or path can be specified:
 
-```bash
-# Create a virtual environment at `my-name`
+```console
+$ # Create a virtual environment at `my-name`
 uv venv my-name
```
Comment (eth3lbert) [inline docs/pip/environments.md:23]:
``` suggestion
$ # Create a virtual environment at `my-name`
$ uv venv my-name
```

File: docs/pip/environments.md [hunk 3]
```diff
@@ -11,22 +11,22 @@ installation's environment. Unlike `pip`, uv requires using a virtual environmen
...
 ```
 
 A Python version can be requested:
 
-```bash
-# Create a virtual environment with Python 3.11
+```console
+$ # Create a virtual environment with Python 3.11
 uv venv --python 3.11
```
Comment (eth3lbert) [inline docs/pip/environments.md:30]:
``` suggestion
$ # Create a virtual environment with Python 3.11
$ uv venv --python 3.11
```

File: docs/pip/environments.md [hunk 4]
```diff
@@ -39,17 +39,17 @@ documentation for more details.
...
 uv pip install ruff
 ```
 
 The virtual environment can be "activated" to make its packages available:
 
-```bash
-# On macOS and Linux.
+```console
+$ # On macOS and Linux.
 source .venv/bin/activate
```
Comment (eth3lbert) [inline docs/pip/environments.md:53]:
``` suggestion
$ # On macOS and Linux.
$ source .venv/bin/activate
```

File: docs/pip/packages.md [hunk 5]
```diff
@@ -4,50 +4,50 @@
...
+$ uv pip install "git+https://github.com/astral-sh/ruff"
 ```
 
 To install a package from GitHub at a specific reference:
 
-```bash
-# Install a tag
+```console
+$ # Install a tag
 uv pip install "git+https://github.com/astral-sh/ruff@v0.2.0"
```
Comment (eth3lbert) [inline docs/pip/packages.md:51]:
```suggestion
$ # Install a tag
$ uv pip install "git+https://github.com/astral-sh/ruff@v0.2.0"
```

File: docs/guides/integration/alternative-indexes.md [hunk 6]
```diff
@@ -19,8 +19,8 @@ pipeline](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variabl
 credentials can be provided via the "Basic" HTTP authentication scheme. Include the PAT in the
 password field of the URL. A username must be included as well, but can be any string.
 
-```bash
-# With the token stored in the `ADO_PAT` environment variable
+```console
+$ # With the token stored in the `ADO_PAT` environment variable
 export UV_EXTRA_INDEX_URL=https://dummy:$ADO_PAT@pkgs.dev.azure.com/{organisation}/{project}/_packaging/{feedName}/pypi/simple/
```
Comment (eth3lbert) [inline docs/guides/integration/alternative-indexes.md:24]:
``` suggestion
$ # With the token stored in the `ADO_PAT` environment variable
$ export UV_EXTRA_INDEX_URL=https://dummy:$ADO_PAT@pkgs.dev.azure.com/{organisation}/{project}/_packaging/{feedName}/pypi/simple/
```

File: docs/guides/integration/alternative-indexes.md [hunk 7]
```diff
@@ -43,8 +43,8 @@ mode](https://github.com/astral-sh/uv/blob/main/PIP_COMPATIBILITY.md#registry-au
 The `keyring` CLI requires a username in the URL, so the index URL must include the default username
 `VssSessionToken`.
 
-```bash
-# Pre-install keyring and the Artifacts plugin from the public PyPI
+```console
+$ # Pre-install keyring and the Artifacts plugin from the public PyPI
```
Comment (eth3lbert) [inline docs/guides/integration/alternative-indexes.md:47]: ditto.

**Comment by zanieb** [comment]
@eth3lbert thank you! I fixed those, though I mostly tried to move content outside of the codeblock.

**Comment by eth3lbert** [comment]
@zanieb You should probably also add a `$` at the beginning of `#` comments.

While serving this locally, I noticed it transforms the code block into the following on the website (http://localhost:8000/uv/guides/integration/alternative-indexes/#using-keyring):
```html
<code tabindex="0"><a id="__codelineno-1-1" name="__codelineno-1-1" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-1"></a><span class="gp"># </span>Pre-install<span class="w"> </span>keyring<span class="w"> </span>and<span class="w"> </span>the<span class="w"> </span>Artifacts<span class="w"> </span>plugin<span class="w"> </span>from<span class="w"> </span>the<span class="w"> </span>public<span class="w"> </span>PyPI
<a id="__codelineno-1-2" name="__codelineno-1-2" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-2"></a><span class="gp">$ </span>uv<span class="w"> </span>tool<span class="w"> </span>install<span class="w"> </span>keyring<span class="w"> </span>--with<span class="w"> </span>artifacts-keyring
<a id="__codelineno-1-3" name="__codelineno-1-3" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-3"></a>
<a id="__codelineno-1-4" name="__codelineno-1-4" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-4"></a><span class="gp"># </span>Enable<span class="w"> </span>keyring<span class="w"> </span>authentication
<a id="__codelineno-1-5" name="__codelineno-1-5" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-5"></a><span class="gp">$ </span><span class="nb">export</span><span class="w"> </span><span class="nv">UV_KEYRING_PROVIDER</span><span class="o">=</span>subprocess
<a id="__codelineno-1-6" name="__codelineno-1-6" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-6"></a>
<a id="__codelineno-1-7" name="__codelineno-1-7" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-7"></a><span class="gp"># </span>Configure<span class="w"> </span>the<span class="w"> </span>index<span class="w"> </span>URL<span class="w"> </span>with<span class="w"> </span>the<span class="w"> </span>username
<a id="__codelineno-1-8" name="__codelineno-1-8" href="http://localhost:8000/uv/guides/integration/alternative-indexes/#__codelineno-1-8"></a><span class="gp">$ </span><span class="nb">export</span><span class="w"> </span><span class="nv">UV_EXTRA_INDEX_URL</span><span class="o">=</span>https://VssSessionToken@pkgs.dev.azure.com/<span class="o">{</span>organisation<span class="o">}</span>/<span class="o">{</span>project<span class="o">}</span>/_packaging/<span class="o">{</span>feedName<span class="o">}</span>/pypi/simple/
</code>
```

And it shows that the `#` comment has the `gp` classname, which could cause issues if copied, as the `#` would be ignored!

**Comment by zanieb** [comment]
😭 that's tragic.

**Comment by eth3lbert** [comment]
Perhaps adding a note to `STYLE.md`? Alternatively, modify the copy logic to strictly ignore just `$`, not all `go` or `gp` classes.

**Pull request merged**